udmis/src/main/java/com/google/bos/udmi/service/core/ReflectProcessor.java

```diff
@@ -93,6 +93,7 @@ private CloudModel getReflectionResult(Envelope attributes, Map<String, Object>
   }
 
   private void processException(Envelope reflection, Exception e) {
+    debug("Processing exception: " + GeneralUtils.friendlyStackTrace(e));
```

Comment:
This is explicitly NOT logging it in the standard way (i.e. the "friendlyStackTrace" method), so I don't want the entire stack trace here.  Most underlying implementations either 1) don't have a specific way to log an Exception since they're not Java-specific (e.g. cloud logging), or 2) Will blindly log the entire stack-trace.  In this case, it just makes the whole output too messy; the stack trace itself is available elsewhere.